TwilioVerifySDK/TwilioVerify/Sources/Domain/Factor/FactorFacade.swift

```diff
@@ -140,12 +148,23 @@ extension FactorFacade {
       self.clearStorageOnReinstall = clearStorageOnReinstall
       return self
     }
-    
+
+    public func setAccessGroup(_ accessGroup: String?) -> Self {
+      self.accessGroup = accessGroup
+      return self
+    }
+
+    public func setUserDefaults(_ userDefaults: UserDefaults) -> Self {
+      self.userDefaults = userDefaults
+      return self
+    }
+
     func build() throws -> FactorFacadeProtocol {
       let factorAPIClient = FactorAPIClient(networkProvider: networkProvider, authentication: authentication, baseURL: url)
-      let secureStorage = SecureStorage()
-      let factorMigrations = FactorMigrations()
-      let storage = try Storage(secureStorage: secureStorage, migrations: factorMigrations.migrations(), clearStorageOnReinstall: clearStorageOnReinstall)
+      let keychainQuery = KeychainQuery(accessGroup: accessGroup)
+      let secureStorage = SecureStorage(keychain: keychain, keychainQuery: keychainQuery)
+      let migrations = FactorMigrations().migrations()
+      let storage = try Storage(secureStorage: secureStorage, keychain: keychain, userDefaults: userDefaults, migrations: migrations, clearStorageOnReinstall: clearStorageOnReinstall, accessGroup: accessGroup)
```

Comment:
<table data-meta="generated_by_UnitTests">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/twilio/twilio-verify-ios/blob/70038a94414b49a629b568bae8208ea1d6186aba/TwilioVerifySDK/TwilioVerify/Sources/Domain/Factor/FactorFacade.swift#L167"></span>Line should be 195 characters or less: currently 209 characters<br />
<code>line_length</code> <code>FactorFacade.swift:167</code></td>
    </tr>
  </tbody>
</table>